src/modules/dashboard/components/ColonyHome/ColonyHome.tsx

```diff
@@ -220,21 +240,79 @@ const ColonyHome = ({
     />
   );
 
+  const renderNewTaskButton = (
+    <>
+      {canCreateTask && (
+        <ActionButton
+          button={({ onClick, disabled, loading }) => (
+            <Button
+              appearance={{ theme: 'primary', size: 'medium' }}
+              text={MSG.newTaskButton}
+              disabled={disabled}
+              loading={loading}
+              onClick={throttle(onClick, 2000)}
+            />
+          )}
+          disabled={isInRecoveryMode}
+          error={ActionTypes.TASK_CREATE_ERROR}
+          submit={ActionTypes.TASK_CREATE}
+          success={ActionTypes.TASK_CREATE_SUCCESS}
+          transform={transform}
+          loading={isTaskBeingCreated}
+        />
+      )}
+    </>
+  );
+
+  const crumbs = (domains || [])
+    .sort((a, b) => a.id - b.id)
+    .reduce(
+      (accumulator, element) => {
+        if (element.id <= filteredDomainId && element.name) {
+          const message = {};
+          message[element.name] = {
+            id: element.name,
+            defaultMessage: element.name,
+          };
+          const messageDescriptor = defineMessages(message);
+          accumulator.push(messageDescriptor);
+        }
+        return accumulator;
+      },
+      [defineMessages({ root: { id: 'root', defaultMessage: 'root' } })],
+    );
+
   return (
     <div className={styles.main}>
       <aside className={styles.colonyInfo}>
-        <ColonyMeta
-          colony={colony}
-          canAdminister={!isInRecoveryMode && canAdminister(permissions)}
-        />
+        <div className={styles.metaContainer}>
+          <ColonyMeta
+            colony={colony}
+            canAdminister={!isInRecoveryMode && canAdminister(permissions)}
+          />
+        </div>
+        <div className={styles.domainContainer}>
+          <ColonyDomains
```

Comment:
I just noticed this:

`ColonyDomains` need to be refactored to be part of the `ColonyMeta` component, not external to it

Reply:
@rdig Could you tell me a reason or an argument to cause more work?

Reply:
Also I just realised a little icon I missed in the DomainsList I assume, it's to collapse the list, so it will have it's own state, saying that I would say it's better to be in it's own component. 
<img width="75" alt="Screen Shot 2019-09-19 at 14 18 02" src="https://user-images.githubusercontent.com/6294044/65243342-757e7380-dae8-11e9-94fa-c64610a41f6a.png">

Reply:
> @rdig Could you tell me a reason or an argument that justifies this, because it is causing more work.

Because both from a logical point of view, and from a code point of view they're part of the meta information regarding a colony.

From a code point of view, after they're moved into the sidebar component (aka: Colony Meta), you actually simply a bunch of styles, as they can now be applied globally (you won't need to individually set margins and padding on sections)

> because it is causing more work.

If you don't want to do it here, I'll do it in my downstream branch #1839, but either way this is happening

Reply:
> Also I just realised a little icon I missed in the DomainsList I assume, it's to collapse the list, so it will have it's own state, saying that I would say it's better to be in it's own component.

I'm not 100% sure that is to collapse / show the list. It might be there to add new domains strait from the new Colony Home.

Better ask @JoinColony/product to make sure

Reply:
please ignore this icon

Reply:
> If you don't want to do it here, I'll do it in my downstream branch #1839, but either way this is happening

I'll do it just wanted to understand the reasoning behind.